demisto_sdk/commands/common/handlers/yaml/ruamel_handler.py

```diff
@@ -26,7 +26,7 @@ def dump(self, data, stream, sort_keys=False):
             data = order_dict(data)
         self._yaml.dump(data, stream)
 
-    def dumps(self, data, sort_keys=False):
+    def dumps(self, data, sort_keys=False, indent=None):
```

Comment:
if possible, please maintain a constant order of arguments (see `xsoar_handler.dumps` - there, `indent` comes before `sort_keys`)